tool/tctl/common/lock_command_test.go

```diff
@@ -66,12 +66,12 @@ func TestLocks(t *testing.T) {
 			},
 			Message: "Come see me",
 		})
-		expected.SetCreatedBy("admin")
 		require.NoError(t, err)
+		expected.SetCreatedBy("admin")
 
-		dt, err := time.Parse(time.DateTime, "1984-04-04 00:00:00")
+		dt, err := time.Parse(time.DateTime, "2006-01-02 15:04:05")
```

Comment:
If you change `withFakeClock` to accept a clock, then you could just use `Now()` instead of having to use this magic string that must match the timestamp in another file.:
```
clock := clockwork.NewFakeClock()
makeAndRunTestAuthServer(..., withFakeClock(clock))

start := clock.Now()
expected.SetCreatedAt(now)
runLockCommand(...)
```

Reply:
Good call, updated!